ext/dapr-ext-workflow/setup.cfg

```diff
@@ -25,7 +25,7 @@ packages = find_namespace:
 include_package_data = True
 install_requires =
     dapr-dev >= 1.11.0rc1.dev
-    durabletask >= 0.1.0
+    durabletask >= 0.1.1a1
```

Comment:
You really shouldn't use alpha versions as dependencies in this SDK. Please use a proper versioning scheme on the durable task side :)

Reply:
Will update once a stable version is released